src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -147,6 +155,31 @@ let require=___nfyCreateRequire(import.meta.url);
       srcFile,
       outputExtension,
     })
+
+    for (const [filename] of Object.entries(metafile.outputs)) {
+      const content = await readFile(filename, { encoding: 'utf-8' })
```

Comment:
Rather than esbuild writing to disk and then us reading + rewriting the files, you could [set `write: false`](https://esbuild.github.io/api/#write) to make it return the contents, which you can then patch and write to disk just once.

On the other hand, the current approach feels less intrusive, in the sense that (hopefully) a fix for this is added upstream and we can just remove this whole block without changing the way files are written.

No strong opinion, just wanted to present that option.

Reply:
I like that the current approach is less intrusive. Let's stick with that for now.